### ISSUE TITLE
Temperature key?

### ISSUE THREAD

**Issue by adelin-mcbsoft** [modest/ghn-tweak#1]
Hi,

You've mentioned in _readme.md_ that you can read the (CPU's) temperature...
Could you please tell me which is the key for it?

I have looked through all the entries found in /getall.html and couldn't identify it.
My HW_PRODUCT is DCP962P.

Thank you,
A.

**Comment by modest** [comment]
Are you using authenticated cookies when requesting /getall.html ? You get a subset of values when unauthenticated.

**Comment by adelin-mcbsoft** [comment]
Yes, while accessing the interface through the browser for example, you cannot request `/getall.html` without being authenticated. I get a `HTTP 403 Forbidden` page if I try to.

But after accessing the page (preceding authentication), I see all sorts of keys, but cannot find one related to the temperature, that's why I was curious if you could tell me which one the key is (or should be).

Thanks.

**Comment by lazna** [comment]
for me, both get.html and getall.html produce empty page. When displaying source, there is a 33 lines of MaxLinear legal notice (formated as a HTML comment), and three more empty lines. Nothing else.

What does this means? My device (Firstmile G4201C) have modyfied firmware which does not produce raw data?

**Comment by ensonic** [comment]
For me the key is `TEMPSENSORS.GENERAL.MEASURE` I have not figured yet how to map the value to °C or F.
